uint16 motor_id
uint16 motor_p